apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -190,6 +218,37 @@ export const InstructorSharing = ({
                     </form>
                   </td>
                 </tr>
+                <tr>
+                  <th>Public Questions Page</th>
+                  <td class="form-inline align-middle">
+                    <span class="input-group">
+                      <span readonly class="form-control form-control-sm"
+                        >${publicSharingLink}</span
+                      >
+                      <div class="input-group-append">
+                        <button
+                          type="button"
+                          class="btn btn-sm btn-outline-secondary btn-copy"
+                          data-clipboard-text="${publicSharingLink}"
+                          aria-label="Copy public sharing link"
+                        >
+                          <i class="far fa-clipboard"></i>
+                        </button>
+                        <button
+                          type="button"
+                          title="Public Questions Link QR Code"
+                          aria-label="Public Questions QR Code"
+                          class="btn btn-sm btn-outline-secondary js-public-sharing-link-qrcode-button"
+                        >
+                          <i class="fas fa-qrcode"></i>
+                        </button>
+                        <div class="d-none">
+                          <div id="js-public-sharing-link-qrcode">${publicSharingLinkQRCode}</div>
```

Comment:
For some reason, this QR code SVG is not getting un-escaped properly, and it ends up just being text in the browser instead of rendering as SVG. Does this have something to do with something I don't understand about the prairielearn/html package?

Reply:
QR code removed in 0385fb4.